cmds/wifi/wifi.go

```diff
@@ -16,11 +16,10 @@ import (
 const (
 	cmd          = "wifi [options] essid [passphrase]"
 	nopassphrase = `network={
-	ssid="%s"
-	proto=RSN
-	key_mgmt=NONE
-}
-`
+		ssid="%s"
```

Comment:
are you sure this change is safe? did you make sure that the wifi supplicant will parse it with this added space?

Reply:
Tested it out on the Chromebook. The above is safe